### PR TITLE
Story#1066 attributes of session list

### PR DESCRIPTION
**Pull request by nevil-mathew**
Story: [Story#1066](https://katha.shikshalokam.org/story-view-1066.html)
Bases on tasks from [Sheet](https://docs.google.com/spreadsheets/d/1Qk24NxKg1TXvjxCcvJksOUpglzj2NibhfMB3hmPiK-w/edit#gid=1699939388)

### CODE REVIEW

File: src/database/queries/sessions.js [hunk 1]
```diff
@@ -259,6 +259,7 @@ exports.findAllSessions = async (page, limit, search, filters) => {
 				'image',
 				'created_at',
 				'meeting_info',
+				'created_by',
```
Comment (rakeshSgr) [inline src/database/queries/sessions.js:262]: @nevil-mathew we need mentor_name right ?

Comment (nevil-mathew) [reply src/database/queries/sessions.js:262]: we're not using this function for the list apis @rakeshSgr

File: src/services/sessions.js [hunk 2]
```diff
@@ -43,21 +46,41 @@ module.exports = class SessionsHelper {
...
 	static async create(bodyData, loggedInUserId, orgId) {
-		bodyData.mentor_id = loggedInUserId
 		try {
-			const mentorDetails = await mentorExtensionQueries.getMentorExtension(loggedInUserId)
+			bodyData.created_by = loggedInUserId
+			bodyData.updated_by = loggedInUserId
+
+			const mentorIdToCheck = bodyData.mentor_id ? bodyData.mentor_id : loggedInUserId
+
+			const mentorDetails = await mentorExtensionQueries.getMentorExtension(mentorIdToCheck)
```
Comment (rakeshSgr) [inline src/services/sessions.js:55]: @nevil-mathew we can remove this check, once roles and permissions are enabled.

File: src/services/sessions.js [hunk 3]
```diff
@@ -1378,4 +1419,277 @@ module.exports = class SessionsHelper {
...
+	 * @param {string} userId - User ID of the creator.
+	 * @param {Object} queryParams - Query parameters for filtering sessions.
+	 * @param {string} timezone - Time zone for date and time formatting.
+	 * @param {string} searchText - Text to search for in session titles.
+	 * @returns {Promise<Object>} - A promise that resolves to a response object containing
+	 *                             a CSV stream of the session list for download.
+	 * @throws {Error} - Throws an error if there's an issue during processing.
+	 */
+
+	static async downloadList(userId, queryParams, timezone, searchText) {
```
Comment (rakeshSgr) [inline src/services/sessions.js:1435]: @nevil-mathew add sorting

Comment (nevil-mathew) [reply src/services/sessions.js:1435]: added

**Pull request merged**